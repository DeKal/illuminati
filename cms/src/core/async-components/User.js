import loadable from '@loadable/component'

const User = loadable(() =>
  import(/* webpackChunkName: "user" */ 'user/components/User')
)

export default User
