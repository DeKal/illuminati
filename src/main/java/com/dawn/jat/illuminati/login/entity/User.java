package com.dawn.jat.illuminati.login.entity;

import lombok.Data;
import lombok.Generated;
import lombok.NoArgsConstructor;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Generated
@NoArgsConstructor
@RequiredArgsConstructor
@Data
@Document(collection = "users")
public class User {
    @Id
    private String id;
    @NonNull
    private String username;
    @NonNull
    private String password;
    @NonNull
    private String role;
}
