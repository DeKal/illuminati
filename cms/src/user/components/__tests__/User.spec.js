import * as React from 'react'
import { shallow } from 'enzyme'
import User from '../User'

describe('User', () => {
  it('should render User with no error', function() {
    const wrap = shallow(<User />)
    expect(wrap).toMatchSnapshot()
  })
})
