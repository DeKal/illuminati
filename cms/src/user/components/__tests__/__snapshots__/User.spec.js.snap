// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should render User with no error 1`] = `
<div>
  <h2>
     User
  </h2>
</div>
`;
